example_data/epilepsy_bundle.json

```diff
@@ -0,0 +1,656 @@
+      "resource": {
+        "resourceType": "DiagnosticReport",
+        "id": "43a930ca-d757-11ec-9d64-0242ac120002",
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "81247-7",
+              "display": "Early onset or syndromic epilepsy"
+            }
+          ]
+        },
+        "subject": {
+          "type": "Patient",
+          "reference": "Patient/43a92b3e-d757-11ec-9d64-0242ac120002"
+        },
+        "performer": [
+          {
+            "type": "Practitioner",
+            "reference": "Practitioner/43a9517c-d757-11ec-9d64-0242ac120002"
+          }
+        ],
+        "specimen": [
+          {
+            "type": "Specimen",
+            "reference": "Specimen/43a96978-d757-11ec-9d64-0242ac120002"
+          }
+        ],
+        "conclusion": "Confirms the diagnosis of Childhood-nset epileptic encephalopathy (EEOC)."
+      },
+      "request": {
+        "method": "POST",
+        "url": "DiagnosticReport"
+      }
+    },
+    {
+      "fullUrl": "Observation/43a9579e-d757-11ec-9d64-0242ac120002",
+      "resource": {
+        "resourceType": "Observation",
+        "id": "43a9579e-d757-11ec-9d64-0242ac120002",
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "69548-6",
+              "display": "Genetic variant accesment"
+            }
+          ]
+        },
+        "meta": {
+          "profile": [
+            "http://hl7.org/fhir/uv/genomic-reporting/gene-panel"
+          ]
+        },
+        "subject": {
+          "type": "Patient",
+          "reference": "Patient/43a92b3e-d757-11ec-9d64-0242ac120002"
+        },
+        "specimen": {
+          "type": "Specimen",
+          "reference": "Specimen/43a96978-d757-11ec-9d64-0242ac120002"
+        },
+        "performer": [
+          {
+            "type": "Practitioner",
+            "reference": "Practitioner/43a9517c-d757-11ec-9d64-0242ac120002"
+          }
+        ],
+        "component": [
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "48013-7",
+                  "display": "HGVS Description"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://www.ncbi.nlm.nih",
+                  "code": "gene_panel",
+                  "display": "NM_001271.3: c.3917del p.(Gln1306Argfs*13)"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "53034-5",
+                  "display": "zygosity"
```

Comment:
This is based on what is in the report. "display": "zygosity" I would have to search the proper loinc / snomed code

Reply:
Ah right, I'm following you - are you fine if I pull through the display value from the loinc code? We'd like to at least keep the naming consistent with the coding system we're using so that it's clear